.travis.yml

```diff
@@ -1,2 +1,14 @@
 language: clojure
+before_install:
+  # Install mono
+  - sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+  - sudo apt install apt-transport-https ca-certificates
+  - echo "deb https://download.mono-project.com/repo/ubuntu stable-xenial main" | sudo tee /etc/apt/sources.list.d/mono-official-stable.list
```

Comment:
Does the Travis image not come with mono, or is there not a package for it in the distro repositories?

Reply:
There doesn't appear to be either unfortunately. Travis CI comes with `mono` for C# and F# projects but not for Clojure. I also tried looking into a "multi-language" approach, but this functionality doesn't seem to be supported. I checked the `snap` package manager as well and didn't find `mono`. As such I think this is currently the least painful solution.